lib/shared/config-manager/src/index.ts

```diff
@@ -184,10 +291,32 @@ export class EnvironmentConfigManager {
                 `Failed to download config, using cached version. url: ${url}.`,
             )
         } else if (responseError?.status === 403) {
-            this.stopPolling()
+            this.cleanup()
             throw new UserError(`Invalid SDK key provided: ${this.sdkKey}`)
         } else {
             throw new Error('Failed to download DevCycle config.')
         }
     }
+
+    private handleSSEConfig(projectConfig: string) {
+        if (this.enableRealtimeUpdates) {
+            const configBody = JSON.parse(projectConfig) as ConfigBody<string>
+            const originalConfigSSE = this.configSSE
+            this.configSSE = configBody.sse
+
+            // Reconnect SSE if not first config fetch, and the SSE config has changed
```

Comment:
Does this reconnecting logic make sense?

Reply:
How can SSE be "started" before there is a config downloaded? Might make sense to track the state similar to go?

Reply:
this function gets called during the processing of the config response, including the first config download.